src/Kuzzle.js

```diff
@@ -203,6 +203,14 @@ class Kuzzle extends KuzzleEventEmitter {
     return this.protocol.sslConnection;
   }
 
+  isLoggued () {
+    if (!this.jwt) {
+      return false;
+    }
+
+    return this.jwtExpiresAt > Date.now();
```

Comment:
This falls short as soon as the `jwt` property is set with a custom token (i.e. saved from an earlier connection), which is a very, very common situation.

For this to work, you must also update the `jwt` setter, decode the token's payload and retrieve its timeout value.